fitbenchmarking/fitbenchmarking
# Added errorbars to the plots in the fitting report

## Description of work

Fixes #1245 

<img width="945" alt="image" src="https://github.com/fitbenchmarking/fitbenchmarking/assets/47083562/294136e9-18aa-4915-89d7-9b2b9505436e">

<img width="953" alt="image" src="https://github.com/fitbenchmarking/fitbenchmarking/assets/47083562/9b23de89-a46f-4856-98e3-d26bcc52a996">





## Testing Instructions



1. Run `fitbenchmarking` from the cli. Once benchmarking is complete, navigate to the fitting report pages and verify error bars are present in the plots.
2. Run `pytest fitbenchmarking/results_processing/tests/test_plots.py --test-type default` to verify tests are passing.

## Checklist
